lib/atomic_web/components/dropdown.ex

```diff
@@ -0,0 +1,33 @@
+defmodule AtomicWeb.Components.Dropdown do
+  @moduledoc false
+  use AtomicWeb, :component
```

Comment:
```suggestion
  use Phoenix.Component
```
Ask @joaodiaslobo why :milady: